File: week-03/day-3/07.py
# create a function that takes a list and returns a new list with all the elements doubled
class Doublelist():

    def add(self, list):
        self.list = list

    def size(self):
        j=0
        for i in self.list:
            j+=1
        return j

    def double(self):
        doublelist = []
        i = 0
        while i < self.size():
            doublelist+= [self.list[i]*2]
            i+= 1
        return doublelist

items = (1,2,3,4,5,6,7,8,9)
first = Doublelist()
first.add(items)
print(first.double())

File: week-07/day-3/07.js
'use strict';

var numbers = [2, 5, 11, 31];
//true
//[Finished in 931.127s]
// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function ifprime(arr) {

  function isprime(item){
    for (var i = 2; i < item; i++){
      if (item % i == 0){
        return false;
      }
    }
    return true;
    }
  return arr.every(isprime)
}
console.log(ifprime(numbers));
